triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -85,8 +85,8 @@ object TriggerRunnerImpl {
             replyTo ! QueryingACS
             Behaviors.same
           case QueryACSFailed(cause: io.grpc.StatusRuntimeException)
-              if cause.getStatus == io.grpc.Status.UNAUTHENTICATED =>
-            throw new UnauthenticatedException(s"Querying ACS failed: ${cause.toString}")
+              if cause.getStatus.getCode == Code.UNAUTHENTICATED =>
```

Comment:
Just double-checking that we have tests for this.

Reply:
Not sure, I haven't found one now. There was a test that was failing due to this kind of comparison. However, I think we would need to revisit the test coverage for the whole class.